test/test_checks.py

```diff
@@ -34,6 +34,18 @@ def test_noqa_doubles(self):
         self.assertEqual(list(checker.run()), [])
 
 
+class DoublesMixedTestChecks(TestCase):
```

Comment:
Nah, it should be fine for here. #27 should be landed soon and then all multiline alterations belong here =P